source/Conversation.cpp

```diff
@@ -256,6 +265,16 @@ void Conversation::Save(DataWriter &out) const
 				out.EndChild();
 				continue;
 			}
+			if(!node.actions.IsEmpty())
+			{
+				out.Write("action", TokenName(node.data[0].second));
```

Comment:
Does the action keyword need a TokenName? It doesn't seem to be read during loading.

Reply:
This is the same behavior as branch and apply. Instead of needing a `goto <label>` (which is something I pointed out about the prior implementation), the conversation understands the second argument here as the goto location

Reply:
I'm not too familiar with this part of the Conversations code and the code looked a bit strange, but I just looked at `branch` and `apply` and they look equally strange so for now I will just believe you that this code here is correct.